### DISCUSSION TITLE
2min insignia change for more balanced pvp

### DISCUSSION THREAD

**Discussion by Vitusin** [chromiecraft/chromiecraft#3523] [Suggestions]
There is one thing that is not talked about much and that is the imbalance in insignia 5min for nonhumans VS human racial 2min. I think the time difference speaks for itself. I'd love it if pvp active players would speak up and give their opinion here.  I'm obviously up for a 2min change. Thanks everyone for the replies.

**Comment by KaneoHunter** [comment]
play human then lol

**Reply by Vitusin** [reply]
I am. Pyrofobia. But i am even playing as hunter(Vitusin) or druid(Leaf)and bcs of that i see the differenc and what to make it right.

**Reply by KaneoHunter** [reply]
So you want to change the balance of the game because you can't pick Human on your busted hunter or your busted druid? lol

Before looking at making this a custom game with custom items, let's instead look at % balancing specific classes in pvp. Speaking of which, there should be a bigger blanket damage reduction so it's not a 1-shot fest, but apparently people like that so OMEGALUL kill your own game mode I guess pvp community.

**Reply by Vitusin** [reply]
> So you want to change the balance of the game because you can't pick Human on your busted hunter or your busted druid? lol
> 
> Before looking at making this a custom game with custom items, let's instead look at % balancing specific classes in pvp. Speaking of which, there should be a bigger blanket damage reduction so it's not a 1-shot fest, but apparently people like that so OMEGALUL kill your own game mode I guess pvp community.

Well my point is human racial was made with time when every other class have acces to 2min insignia.

**Reply by haebawari** [reply]
> Well my point is human racial was made with time when every other class have acces to 2min insignia.

yes, and wrath talents were made with time when every class has access to lvl 80, your point?

**Reply by Vitusin** [reply]
> > Well my point is human racial was made with time when every other class have acces to 2min insignia.
> 
> yes, and wrath talents were made with time when every class has access to lvl 80, your point?

Thats just ,,whataboutism" i am talking about whats fair.. you r playing ?

**Reply by haebawari** [reply]
> i am talking about whats fair

Yes, killing mages by targeting them with BM pet and watching them being unable to do anything because they don't have Invisibility or Mirror Image yet is also fair, right? Should we maybe try to remove this unfairness from the bracket as well?

**Reply by oterfesk** [reply]
you do see the difference between giving classes whole ass abilities and slightly tuning a cooldown for an already existing item
.. right..?

**Comment by caloides** [comment]
I believe this would make it fair in BGs, right now we are at a big disadvantage compared to the human race

**Comment by haebawari** [comment]
That would nullify half the reason to pick humans.
Also think about this: hunter is asking you to vote to make hunters (and druids) more op. Ugh.

**Reply by Vitusin** [reply]
> That would nullify half the reason to pick humans. Also think about this: hunter is asking you to vote to make hunters (and druids) more op. Ugh.

I am main human mage. But playing even hunter and druid so..

**Reply by haebawari** [reply]
- is a human mage
- constantly pounded by hunts and druids
- "fuckit imma make a hunt and dru myself!"
- rerolls, still sucks
- "REEEEEE NERF HUMANS PLOX"

**Reply by Vitusin** [reply]
> * is a human mage
> * constantly pounded by hunts and druids
> * "fuckit imma make a hunt and dru myself!"
> * rerolls, still sucks
> * "REEEEEE NERF HUMANS PLOX"

Well i see how you think if you r playing anything else then human kek. For me its about fair play. I can play mage, pally or priest any time. Just dont be selfish haebawari, thats all.

**Reply by haebawari** [reply]
> Just dont be selfish

I'm not the one starting a suggestion for a ridiculous change that would directly benefit the character I'm playing the most right now (in PVP).

**Reply by oterfesk** [reply]
yea thats exactly the point of it, to not have people forced into a race they dont wanna play if they also wanna do pvp at the side

**Reply by Vitusin** [reply]
> > Just dont be selfish
> 
> I'm not the one starting a suggestion for a ridiculous change that would directly benefit the character I'm playing the most right now (in PVP).

I want for everyone to have same chances in pvp is it bad ? Or where is the problem you see ?

**Reply by haebawari** [reply]
> Let's give everyone the same chances guys! Don't worry that this change will positively affect all the races that can be hunters (the most broken class in the current bracket) and none of the races that can't be hunters, guys, just don't think about it!

**Reply by Vitusin** [reply]
> > Let's give everyone the same chances guys! Don't worry that this change will positively affect all the races that can be hunters (the most broken class in the current bracket) and none of the races that can't be hunters, guys, just don't think about it!

Well my main is Human mage so. I dont see prob in that. Even as balanc dudu i dont see prot in that. Point is to have fair options.

**Reply by haebawari** [reply]
your "fair options" will make the game even more imbalanced.

**Reply by Vitusin** [reply]
> your "fair options" will make the game even more imbalanced.

My point is what blizz intended. I don't root for blizz because of the way he treated employees, especially women. I'm talking about what was logical, and that was the ballanc between the horde and the alliance. After what happened during TBC and what the que were in BGs it was clear that a change was needed. Radical but not gamechanging. That's why the human racial was created. Since we have 60s now, the game is absolutely unbalanced in pvp because other classes don't have access to 2min insignia. That's all. It's not complicated it's nothing special but it decides who wins BS.

**Reply by haebawari** [reply]
> My point is what blizz intended

Blizz intended for you to play primarily @ lvl 80 on a server with 2 factions, members of which can not join bgs in the same team as members of another one.
Also twinking was a thing in wotlk. If blizz really "intended" then they would have addressed this for all the twinks in wotlk days.

**Reply by Vitusin** [reply]
> 

Blizz intended to balance factions by making humans more attractive(if you played TBC when it was on retail you know how long the que to bg for horde was). I think I've posted this here before, but blizz made this change with the understanding that others would also have access to 2min insignia. My point is that this is unfair and unbalanced. And I don't see what negative impact this would have on the server? More like the only positive. If you see what this threatens or what it would do please share your opinion.

**Comment by Vitusin** [comment]
One point. Just for all human players, please dont vote against. Its just about having a good time in BGs. Be fair.

**Reply by EliteCow09** [reply]
I don't believe this would be a negative change necessarily, so I don't think it would hurt. I'm also pretty used to swapping off the trinket slot when the 5min CD begins at this point, personally, but can see where this has merit. This might help balance some of long stun chains players experience back to back in battlegrounds.

**Comment by RestoCC** [comment]
No

**Reply by Vitusin** [reply]
Can you share your thoughts more ? Why not? How will this negatively affect the pvp scene? Or is this just barking.

**Comment by BeefSteaker** [comment]
Whoever this is, please stop spamming
![image](https://user-images.githubusercontent.com/11332559/171017720-91f823ac-cc7c-4972-b125-de700bbca8be.png)

**Reply by Vitusin** [reply]
Its one per 30min + last few days i am not even playing much bcs IRL. Even talk about that with gms.

**Comment by Dupeloop** [comment]
2min CD would make horde much more appealing to play seeing as this server is very ally dominant

**Comment by Annamaria-CC** [comment]
Thanks for your suggestion. We are always looking for ways to improve, however, considering the mostly negative vote, this is not something we will pursue.

Locking the thread.
